config/environments/development.rb

```diff
@@ -38,4 +38,13 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.paperclip_defaults = {
+      storage: :s3,
+      s3_credentials: {
+        bucket: "cardapp",
+        access_key_id: "AKIAJWQQ5LLD56HNL3BA",
+        secret_access_key: "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",
+        url: ":s3_domain_url"
+      }
+    }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.